CONTRIBUTING.md

```diff
@@ -0,0 +1,55 @@
+# Contributing to this repository
+
+We welcome your contributions! There are multiple ways to contribute.
+
+## Opening issues
+
+For bugs or enhancement requests, please file a GitHub issue unless it's
+security related. When filing a bug remember that the better written the bug is,
+the more likely it is to be fixed. If you think you've found a security
+vulnerability, do not raise a GitHub issue and follow the instructions in our
+[security policy](./SECURITY.md).
+
+## Contributing code
+
+We welcome your code contributions. Before submitting code via a pull request,
```

Comment:
Is contributions to Oracle Products hosted on github typically the trend thats followed?

Reply:
Contributions from external participants are common in Oracle projects and they are welcome. In order for external contributors to be able to contribute, they must sign the [OCA](https://oca.opensource.oracle.com/).